public/locales/de/translation.json

```diff
@@ -72,6 +72,13 @@
         "jobTitle": "Berufsbezeichnung",
         "selectJobTitle": "Wählen Sie eine Berufsbezeichnung",
         "usa": "USA",
+<<<<<<< HEAD
+        "europe": "Europa",
+        "australia": "Australien",
+        "frontEndDeveloper": "front-end developer",
+        "signUpPageText": "Fühlen Sie sich frei, sich anzumelden",
+        "signUp": "Anmelden"
+=======
```

Comment:
You should not have visible Head on  and those lines.
This not resolved. Please keep changes from previous PR and yours.